operator/tests/central/basic/74-assert.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 kind: PersistentVolumeClaim
 metadata:
-  name: stackrox-db
```

Comment:
I think I heard from someone that the `stackrox-db` PVC is still used for some things. If so, please also keep it here.

Reply:
You are right, it makes sense to check for both central-db and stackrox-db PVCs